homework/index.js

```diff
@@ -21,27 +21,117 @@
     parent.appendChild(elem);
     Object.keys(options).forEach(key => {
       const value = options[key];
-      if (key === 'text') {
-        elem.innerText = value;
+      if (key === 'html') {
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
+  function showRepos(leftDiv, repository) {
+    leftDiv.innerHTML = '';
+    const table = createAndAppend('table', leftDiv);
+    const tableBody = createAndAppend('tbody', table);
+    const repositoryTitle = createAndAppend('tr', tableBody);
+    createAndAppend('td', repositoryTitle, { html: 'Repository:', class: 'label' });
+    createAndAppend('a', repositoryTitle, {
+      target: '_blank',
+      href: repository.html_url,
+      html: repository.name,
+    });
+    if (repository.description !== null) {
+      const description = createAndAppend('tr', tableBody);
+      createAndAppend('td', description, {
+        html: 'Description:',
+        class: 'label',
+      });
+      createAndAppend('td', description, { html: repository.description });
+    }
+    const forks = createAndAppend('tr', tableBody);
+    createAndAppend('td', forks, { html: 'Forks:', class: 'label' });
+    createAndAppend('td', forks, { html: repository.forks });
+    const update = createAndAppend('tr', tableBody);
+    createAndAppend('td', update, { html: 'Updated:', class: 'label' });
+    createAndAppend('td', update, { html: repository.updated_at });
```

Comment:
The date as displayed is not very user-friendly. You could research how to convert `updated_at` from a string to a `Date` and then use `.toLocalString()` to convert it to a user-friendly format.